bin/halcyon-build

```diff
@@ -6,6 +6,13 @@ set -e
 [ -d /app/halcyon ] || \
   git clone https://github.com/mietek/halcyon.git /app/halcyon
 
+if [ -z "$HALCYON_AWS_ACCESS_KEY_ID" ] || \
+  [ -z "$HALCYON_AWS_SECRET_ACCESS_KEY" ] || \
+  [ -z "$HALCYON_S3_BUCKET" ]; then
+  printf "Make sure Halcyon environment variables are set.\n" >&2
```

Comment:
We waffled on the `printf` vs `echo` thing in Guides. We landed on `echo` for simple cases. Can you use it here for consistency across the script? (note no trailing `\n` is needed with `echo` too).